### PR TITLE
main/vala: upgrade to 0.44.5

### PR DESCRIPTION
**Pull request by Cogitri**
* Adopt maintership
* Enable tests

### CODE REVIEW

**Comment by Cogitri** [comment]
Times out on armhf on CI :/

File: main/vala/APKBUILD [hunk 1]
```diff
@@ -1,31 +1,50 @@
...
+
+	# https://gitlab.gnome.org/GNOME/vala/issues/809
+	sed '/array-uint8-uchar-compat.vala/d' -i tests/Makefile.am
+
+	autoreconf --install --force
+}
+
 build() {
 	cd "$builddir"
+	export CFLAGS="-O2 -pipe"
```
Comment (ncopa) [inline main/vala/APKBUILD]: I wonder why this is needed?

Comment (Cogitri) [reply main/vala/APKBUILD]: Oh my, good catch. Tried this to debug the one failing test. Will remove :)

Comment (Cogitri) [reply main/vala/APKBUILD]: Fixed.

**Pull request closed without merging**